modules/ws/src/main/java/org/mule/module/ws/consumer/WSConsumer.java

```diff
@@ -111,6 +118,17 @@ public MuleEvent process(MuleEvent event) throws MuleException
     }
 
 
+    private void initializeWsdlRetrievalStrategy()
+    {
+        HttpRequesterConfig httpRequesterConfig = config.getConnectorConfig();
+
+
+        if (httpRequesterConfig == null || httpRequesterConfig.getProxyConfig() == null) {
+            wsdlRetrieverStrategy = new URLRetrieverStrategy();
+        } else {
+            wsdlRetrieverStrategy = new ProxyWsdlRetrieverStrategy(httpRequesterConfig.getTlsContext(), httpRequesterConfig.getProxyConfig(), muleContext, WSConsumer.class.getName());
```

Comment:
Is it needed to send this last parameter? It will always be a fixed value and could use a constant inside ProxyWsdlRetrieverStrategy